tests/end-to-end/director/testctx.go

```diff
@@ -38,7 +38,7 @@ type testContext struct {
 	cli               *gcli.Client
 }
 
-const defaultScopes = "runtime:write application:write label_definition:write integration_system:write"
+const defaultScopes = "application_template:write runtime:write application:write label_definition:write integration_system:write"
```

Comment:
what about `application_template:read`? Why don't we use it?

Reply:
Because we need to create application template in tests.